prdoc/pr_5372.prdoc

```diff
@@ -0,0 +1,65 @@
+title: "elastic scaling: add claim queue offset to cumulus"
+
+doc:
+  - audience: [Node Dev, Runtime Dev]
+    description: |
+      Add a claim queue offset storage item to the parachain-system pallet and a runtime API for querying it.
+      Also use the claim queue offset and the claim queue relay chain runtime API in the slot based collator (if available)
+      to determine which cores to build on.
+      Part of implementing https://github.com/polkadot-fellows/RFCs/pull/103.
+
+crates:
+  - name: cumulus-client-consensus-aura
+    bump: minor
+  - name: cumulus-relay-chain-inprocess-interface
+    bump: minor
+  - name: cumulus-relay-chain-interface
+    bump: major
+  - name: cumulus-relay-chain-minimal-node
+    bump: none
+  - name: cumulus-relay-chain-rpc-interface
+    bump: minor
+  - name: cumulus-pallet-parachain-system
+    bump: minor
+  - name: asset-hub-rococo-runtime
+    bump: minor
+  - name: asset-hub-westend-runtime
+    bump: minor
+  - name: bridge-hub-rococo-runtime
+    bump: minor
+  - name: bridge-hub-westend-runtime
+    bump: minor
+  - name: collectives-westend-runtime
+    bump: minor
+  - name: contracts-rococo-runtime
+    bump: minor
+  - name: coretime-rococo-runtime
+    bump: minor
+  - name: coretime-westend-runtime
+    bump: minor
+  - name: glutton-westend-runtime
+    bump: minor
+  - name: people-rococo-runtime
+    bump: minor
+  - name: people-westend-runtime
+    bump: minor
+  - name: seedling-runtime
+    bump: minor
+  - name: shell-runtime
+    bump: minor
+  - name: penpal-runtime
+    bump: minor
+  - name: rococo-parachain-runtime
+    bump: minor
+  - name: polkadot-parachain-bin
+    bump: major
```

Comment:
it's a major bump on the polkadot-parachain-lib because we add a new trait bound to a public trait.
When I initially created this prdoc the polkadot-parachain-lib was part of the bin crate so that's why I bumped the bin crate. Fixed it now